web/src/components/shared/topic-selector.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import Sidebar from './sidebar/sidebar';
+import Accordion from './accordion/accordion';
+import { BodyCopyBoldSmall, BodyCopyRegularSmall } from '../../config/global-styled-components';
+import { colors } from '../../config/global-styles';
+import { device } from '../../config/device';
+
+const TopicSelectorContainer = styled.div`
```

Comment:
Talked with the designer, the font size is okay. she requested if we can make the sidebar bigger for all in desktop view. specifically 595px width. Please make sure it looks good everywhere is used. She also asked if we can change the opacity of the background(when you open the sidebar) to be 0.7 instead of 0.8